_docs/02.3-get_started.markdown

```diff
@@ -45,6 +45,7 @@ $ gulp dev
 
 Now open [localhost:3000](http://localhost:3000) in your browser to access the app. Hello Electrode!
 
+![Hello Electrode-screenshot](http://awesomescreenshot.com/06d66b5aec)
```

Comment:
thanks for the pr! can you please add the image to `/img` folder https://github.com/electrode-io/electrode-io.github.io/tree/develop/img